accessibility/accessibility-in-pdfs.md

```diff
@@ -0,0 +1,92 @@
+# Accessibility in PDFs
+
```

Comment:
As the document is fairly long it would probably be good to add a table of contents. There are several VSCode extensions that do this automatically, I'm happy to commit one myself after we're done with all the comments.